logging/logger_registry_test.go

```diff
@@ -75,26 +75,6 @@ func TestGetRegisteredNames(t *testing.T) {
 	}
 }
 
-func TestRegisterConfig(t *testing.T) {
```

Comment:
This was accidental. I think I used this test as a starting point for my new one.

That said, I don't think this test is covering much ground. So I didn't revert the delete.

Reply:
I think it's the only test we have that directly asserts on the contents of a `Registry`'s [`logConfig`](https://github.com/viamrobotics/rdk/blob/main/logging/logger_registry.go#L16); could we leave it in?

Reply:
Wouldn't a bunch of tests start failing if the contents of a registry's logConfig were incorrect?

An [update is just doing a direct assignment](https://github.com/viamrobotics/rdk/blob/c7926e2680c84a852aa991f49457777a392022f8/logging/logger_registry.go?plain=1#L57). We're missing a lot of tests if that kind of assertion is adding value.